contact.html

```diff
@@ -31,17 +31,19 @@
       <p>Contact me at <span class="email"> michael.dav.kern (at) gmail.com </span>, or drop me a line here:</p>
 
     <form action="form.html">
-    <label>Name</label>
-    <input type="text" name="name" required />
-    <label>Email</label>
+    <label>Name:</label>
```

Comment:
Think about connecting your `<label>`s to your `<input>`s by using the `for` attribute paired with an `id` on the input. It's semantically more correct and also allows for some functionality help such as making a larger click area on mobile devices (for fat thumbs like mine).